src/cockpit/bridge.py

```diff
@@ -154,6 +154,23 @@ def try_to_receive_stderr():
             os.close(stderr_fd)
 
 
+def setup_logging(debug: bool):
+    """Setup our logger with optional filtering of modules if COCKPIT_DEBUG env is set"""
+
+    modules = os.getenv('COCKPIT_DEBUG', '')
+    logging.basicConfig(format='%(name)s-%(levelname)s: %(message)s')
+
+    if debug or modules == 'all':
+        logging.basicConfig(level=logging.DEBUG)
```

Comment:
Any reason why not `logging.root.setLevel`?

Reply:
Fixed.